workshops/05-Refactoring-Workshop/README.md

```diff
@@ -0,0 +1,85 @@
+# Refactoring
+
+You probably have already done some refactoring as part of code katas.
+
+Refactorings are small changes to your code to leave it just a little bit better.
```

Comment:
Think it's worth pulling in a specific definition here, as I hear people use the word "refactoring" all the time when they just mean "changing"!

> Refactoring changes the code to be better, but _leaves the external behaviour the same_.